instrumentation/active_job/opentelemetry-instrumentation-active_job.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'opentelemetry-api', '~> 1.0'
   spec.add_dependency 'opentelemetry-instrumentation-base', '~> 0.21.0'
 
-  spec.add_development_dependency 'activejob', '>= 5.2.0'
+  spec.add_development_dependency 'activejob', '>= 6.0.0', '< 8.0.0'
```

Comment:
Any reason not to optimistically support future activejobs?

Reply:
Personally speaking I think gem requirements send a more explicit signal about what we support and what we don't but I do not have to add that constraint here if you prefer I remove it.

Reply:
🤷 Honestly I don't have any strong feelings either way; I wasn't sure if it was an implicit signal that there may be problems with future versions or something. 😄